sdk/src/main/scala/com/horizen/api/http/ApiResponse.scala

```diff
@@ -17,6 +19,9 @@ trait ErrorResponse extends ApiResponse {
   val exception: Option[Throwable]
 }
 
+trait InternalErrorResponse extends ApiResponse{
```

Comment:
Probably must be inherited from `ErrorResponse`.

Reply:
ok